docs/user-guide/index.md

```diff
@@ -57,6 +57,9 @@ the different pieces:
 * [Connectors](#connectors)
 * [Formatters](#formatters)
 
+## Permission Issue
+If you receive an `EACCES` error when installing `sonar`, the recommended solution is to [change npm's default directory](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory) and then try again. So far, one such permission issue has been reported when user tries to install `sonar` on [Windows Subsystem for Linux](https://msdn.microsoft.com/en-us/commandline/wsl/install_guide). Dependency `canvas-prebuilt` throws an `EACCES` error during the installation process, and this [issue](https://github.com/sonarwhal/sonar/issues/308) was resolved adopting the recommended solution. You can find detailed steps on how to change the npm default directory [here](https://docs.npmjs.com/getting-started/fixing-npm-permissions#option-2-change-npms-default-directory-to-another-directory).
```

Comment:
Split this line into multiple shorter lines, and were possible try to keep the length of the lines to under 80 characters.